File: Java/Warmup-1/lastDigit.java

public class warmup1 {

	// this function return true if the two given not-negative int values have the same last digit
	public boolean lastDigit (int a, int b) {
		
		boolean bool = false;
		String s1 = Integer.toString(a);
		String s2 = Integer.toString(b);
		
		
		if (s1.charAt(s1.length() -1) == s2.charAt(s2.length() -1))
			bool = true;
		
		return bool;

	}
	
	public static void main(String[] args) {
		
		warmup1 o = new warmup1();
		
		System.out.println(o.lastDigit(7, 177777777));
		System.out.println(o.lastDigit(6, 17));
		System.out.println(o.lastDigit(3, 113));

	}
	
}

File: Python/Logic-1/last_digit.py

# given three ints, a b c, return true if two or more of them have the same rightmost digit
# the ints are non-negative
def last_digit(a, b, c):
    bool = False

    if a >= 0 and b >= 0 and c >= 0:
        if a % 10 == b % 10 or b % 10 == c % 10 or a % 10 == c % 10:
            bool = True

    return bool

print(last_digit(23, 19, 13))
print(last_digit(23, 19, 12))
print(last_digit(23, 19 , 3))
